src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -220,14 +235,14 @@ private void signInButtonClicked(ActionEvent e)
         String username = userInput.getText();
         String password = passInput.getText();
 
-        if (ClientUser.isValidInput(username)) {
+        if (ClientUser.isValidInput(username) && ClientUser.isValidInput(password)) {
```

Comment:
Do usernames and passwords have the same requirements? Can passwords only be letters and numbers without spaces? Note that in "real life" that would be a bit of a security risk.